handlers_test.go

```diff
@@ -452,14 +471,46 @@ func TestServerResourcePostHandlerValid(t *testing.T) {
 			assert.Equal(t, rr.Header().Get("Etag"), meta["version"], "ETag and version needs to be the same")
 		})
 	}
+}
+
+func TestServerResourcePostHandlerInvalid(t *testing.T) {
```

Comment:
PP: could be added to the invalid requests block line 150

Reply:
Its testing Invalid Endpoint so I would not add then there